shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/base/BaseExtraSQLITCase.java

```diff
@@ -53,4 +56,16 @@ protected void createTableIndexList() {
     protected void createOrderItemTable() {
         executeWithLog(extraSQLCommand.getCreateTableOrderItem());
     }
+    
+    @Override
+    protected void assertStopScalingSourceWriting() {
+        assertFalse(executeUpdate(extraSQLCommand.getUpdateTableOrderStatus()));
+        assertFalse(executeDDL(extraSQLCommand.getCreateIndexStatus()));
+    }
+    
+    @Override
+    protected void assertRestoreScalingSourceWriting() {
+        assertTrue(executeUpdate(extraSQLCommand.getUpdateTableOrderStatus()));
+        assertTrue(executeDDL(extraSQLCommand.getCreateIndexStatus()));
+    }
```

Comment:
Looks update SQL and index SQL are the same in different databases, these methods could be put in BaseITCase and remove the related SQL configuration to simplify code.

Reply:
Update SQL and Index SQL are slightly different in different databases, can not combine them. 
Take update SQL as an example:
MySQL: ` UPDATE t_order SET status= 'unlock' `
PostgreSQL: ` UPDATE test.t_order SET status= 'unlock' `